packages/client-node/integration/Main.spec.ts

```diff
@@ -36,7 +36,7 @@ describe("Logion SDK", () => {
     });
 
     it("fetches workload", async () => {
-        await workload(state);
+        await workload(state, ALICE, BOB, ALICE);
```

Comment:
Is there any purpose behind listing `ALICE` twice?

Reply:
My intention was to check that the last call to `getWorkload()` does use cached value (i.e. no call to backend), but I did not find an easy way to achieve this. So you're right, as such, it's useless.